WhaleSpotting/react-app/src/styles/Navbar.scss

```diff
@@ -9,6 +9,7 @@
     justify-content: center;
     align-items: center;
     height: $headerHeight;
+    z-index: 1;
```

Comment:
Don't think we need this

Reply:
Without that the card overlaps the header. Strange - only happens with the card and nothing else.

Reply:
As discussed, can be removed